.github/workflows/release.yaml

```diff
@@ -51,4 +51,14 @@ jobs:
           file: Dockerfile
           push: true
           tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
\ No newline at end of file
+          labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
```

Comment:
Can you explain why you chose trivy vs. other vul scanning solutions?

Reply:
Partially experience with bringing pipelines to compliance standards, however from my understanding trivy offers scanning options that others don't use. For example, trivy can also scan a kubernetes environment for misconfigurations, as well as scanning application dependencies, while most simply scan os packages (that's included as well)

Reply:
Awesome! Thanks for the education. I haven't used trivy yet. (My prior experience is with snyk.)